Packs/Base/Scripts/CreateIndicatorRelationship/CreateIndicatorRelationship.py

```diff
@@ -131,11 +131,12 @@ def validate_arguments(args: dict) -> Dict[str, str]:
         raise Exception("entity_b_query can not be used with entity_b and/or entity_b_type")
     if not args.get('entity_b_query') and not args.get('entity_b'):
         raise Exception("Missing entity_b in the create relationships")
-    if (args.get('entity_b') or args.get('entity_b_query')) and not args.get('entity_b_type'):
+    if args.get('entity_b') and not args.get('entity_b_type'):
```

Comment:
`entity_b_type` is not required while using `entity_b_query` argument
![Screen Shot 2022-03-09 at 9 51 30](https://user-images.githubusercontent.com/78307768/157396572-9a2a53c4-b2b4-4c66-9e54-bf0ce9e76ee5.png)